src/Twig/Components/BlogSearchComponent.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Twig\Components;
+
+use App\Entity\Post;
+use App\Repository\PostRepository;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+/**
+ * Live component to display instant search for Posts.
```

Comment:
the file https://github.com/symfony/demo/blob/main/src/Command/AddUserCommand.php describe some usage etc
could be wise to do same here, wdyt?

for ex to https://ux.symfony.com/live-component or/and https://symfony.com/bundles/ux-live-component/current/index.html

Reply:
Good idea indeed. I added some comments in my code!